groups/groups.yaml

```diff
@@ -15,48 +15,20 @@ groups:
       - evan.k.anderson@gmail.com
       - evana@vmware.com
       - evankanderson@knative.team
-      - julz.friedman@uk.ibm.com
-      - julz@knative.team
-      - markusthoemmes@knative.team
-      - markusthoemmes@me.com
-      - me.markusthoemmes@googlemail.com
-      - rhuss@knative.team
-      - rhuss@redhat.com
```

Comment:
@rhuss is an active Knative contributor and a WG lead. Maybe he should be moved from owners to members?!

Reply:
I see other WG leads are in `members`, so, that makes sense.

Reply:
Or let me ask... What's `owners` and what's `members` here?

Owners --> TOC members?
Members --> TOC members + WG leads?

Reply:
Having owners doesn't really matter because it is a mailing list that the public can't join. owners can be moved to members and left empty entirely.

Reply:
Either way, we can't just remove Roland here, as he isn't listed in members. That'll effectively remove him from the list.

So either restore him here, or move everyone to members as suggested.

Reply:
whoops! re-added roland, thanks for catching that

Reply:
@cardil @upodroid Are we good with this one? Sorry for the pressure, but I want to go ahead and work on https://github.com/knative/community/issues/1383